hugegraph-api/src/main/java/org/apache/hugegraph/api/job/AlgorithmAPI.java

```diff
@@ -55,7 +55,7 @@ public class AlgorithmAPI extends API {
     @POST
     @Timed
     @Path("/{name}")
-    @Status(Status.CREATED)
+    @StatusFilter.Status(StatusFilter.Status.CREATED)
```

Comment:
can we search all the "@StatusFilter.Status" and update to "@Status"?

Reply:
yes i will fix it, and notify you to review